src/build.sh

```diff
@@ -124,6 +126,35 @@ function build_kernel_main()
     return "$?"
   fi
 
+  if [[ -n "$from_sha_arg" ]]; then
+    # Check if there is a rebase in process.
+    printf '%s\n' "Verifying if there is a rebase in process..."
+    if [[ -e .git/rebase-merge/done ]]; then
+      printf '%s\n' "ERROR: Abort the repository rebase before continuing with build from sha (use 'git rebase --abort')!"
+      return 22
```

Comment:
Usually, in the kw, we add a comment after the return error that shows the error. For example, return 22 # EINVAL